wagl/standardise.py

```diff
@@ -389,15 +390,35 @@ def card4l(level1, granule, workflow, vertices, method, pixel_quality, landsea,
                                           rori, res_group, compression,
                                           filter_opts, normalized_solar_zenith)
 
-            # metadata yaml's
-            if workflow == Workflow.STANDARD or workflow == Workflow.NBAR:
-                create_ard_yaml(band_acqs, ancillary_group, res_group, normalized_solar_zenith)
-
-            if workflow == Workflow.STANDARD or workflow == Workflow.SBT:
-                create_ard_yaml(band_acqs, ancillary_group, res_group, normalized_solar_zenith, True)
-
             # pixel quality
             sbt_only = workflow == Workflow.SBT
             if pixel_quality and can_pq(level1, acq_parser_hint) and not sbt_only:
                 run_pq(level1, res_group, landsea, res_group, compression, filter_opts, AP.NBAR, acq_parser_hint)
                 run_pq(level1, res_group, landsea, res_group, compression, filter_opts, AP.NBART, acq_parser_hint)
+
+
+        def get_band_acqs(grp_name):
+            acqs = container.get_acquisitions(granule=granule, group=grp_name)
+            nbar_acqs = [acq for acq in acqs if acq.band_type == BandType.REFLECTIVE]
+            sbt_acqs = [acq for acq in acqs if acq.band_type == BandType.THERMAL]
+
+            band_acqs = []
+            if workflow == Workflow.STANDARD or workflow == Workflow.NBAR:
+                band_acqs.extend(nbar_acqs)
+
+            if workflow == Workflow.STANDARD or workflow == Workflow.SBT:
+                band_acqs.extend(sbt_acqs)
+
+            return band_acqs
+
+        # wagl parameters
+        parameters = {'vertices': list(vertices),
+                      'method': method.value,
+                      'rori': rori,
```

Comment:
Keep *rori* for now, as it has been like this for years.  But a TODO for me; provide a clearer and more informative name.